orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/jedis/JedisExecutionRepository.groovy

```diff
@@ -80,26 +96,28 @@ class JedisExecutionRepository implements ExecutionRepository {
 
   JedisExecutionRepository(
     Pool<Jedis> jedisPool,
+    Optional<Pool<Jedis>> jedisPoolPrevious,
     Scheduler queryAllScheduler,
     Scheduler queryByAppScheduler,
     int threadPoolChunkSize
   ) {
     this.jedisPool = jedisPool
+    this.jedisPoolPrevious = jedisPoolPrevious
     this.queryAllScheduler = queryAllScheduler
     this.queryByAppScheduler = queryByAppScheduler
     this.chunkSize = threadPoolChunkSize
   }
 
   @Override
   void store(Orchestration orchestration) {
-    withJedis { Jedis jedis ->
+    withJedis(getJedisPoolForId(orchestration.id)) { Jedis jedis ->
```

Comment:
figure out what redis contains `orchestration.id` (might be `jedisPool` or `jedisPoolPrevious`)